src/hexo/themes/documentation/scripts/external-link-filter.js

```diff
@@ -0,0 +1,18 @@
+/* globals hexo */
+const cheerio = require('cheerio');
+const updateRelAttribute = (data) => {
+    const $ = cheerio.load(data.content, { decodeEntities: false });
+
+    $('a').each(function () {
+        if ($(this).attr('href') && $(this).attr('rel') === 'external') { //eslint-disable-line no-invalid-this
+            $(this).attr('rel', 'noopener noreferrer'); //eslint-disable-line no-invalid-this
```

Comment:
@qzhou1607 We shouldn't overwrite the existing value.

This should be something such as:

```js
$(this).attr('rel', (i, currentValue) => {
    return [...new Set(['noopener', 'noreferrer'].concat(currentValue.split(' ')))].join(' ');
});      
```